compute_endpoint/tox.ini

```diff
@@ -26,7 +26,6 @@ deps = build
        twine
 # clean the build dir before rebuilding
 whitelist_externals = rm
-allowlist_externals = rm
```

Comment:
Vice versa, I think.  `whitelist_externals` is deprecated, right?

https://tox.wiki/en/latest/config.html